changelog.d/2-features/mls

```diff
@@ -0,0 +1,4 @@
+MLS implementation progress:
+ - Remote users can be added to MLS conversations
+ - MLS messages (both handshake and application) are now propagate to remote
```

Comment:
Please don't make it in this PR because of CI: propagate -> propagated. Nothing important, but maybe you feel like fixing it in some other PR or in the release PR.

Reply:
CI doesn't retrigger for changelog-only changes. Fixed the typo now.

Reply:
Or maybe it does? I don't know, I give up understanding concourse. :shrug: